src/runtime_src/core/tools/common/ReportHost.cpp

```diff
@@ -118,15 +118,18 @@ ReportHost::writeReport(const xrt_core::device* /*_pDevice*/,
   Table2D::HeaderData vbnv = {"Shell", Table2D::Justification::right};
   Table2D::HeaderData id = {"Platform UUID", Table2D::Justification::right};
   Table2D::HeaderData instance = {"Device ID", Table2D::Justification::right};
-  std::vector<Table2D::HeaderData> table_headers = {bdf, colon, vbnv, id, instance};
+  Table2D::HeaderData ready = {"Device Ready*", Table2D::Justification::right};

```

Comment:
Try and make these const?

Reply:
Hmm...it looks like you are reviewing one's own PR.